### ISSUE TITLE
InnerBlocks in addition to ACF Fields

### ISSUE THREAD

**Issue by thisanimus** [palmiak/timber-acf-wp-blocks#31]
Is there a way to get both InnerBlocks and ACF fields accessible in the Block editor on a single block?  I'd like to create a block that has some options (background/padding/width) set via ACF fields, and also be able to add InnerBlocks to it.  Currently I have:
```
{#
  Title: Wrapper
  Description: Wrapper
  Category: elevate-blocks
  Icon: admin-comments
  Keywords: link
  Mode: preview
  PostTypes: post page landing-page
  SupportsMode: false
  SupportsMultiple: true
  SupportsAlign: true
  SupportsJSX: true
#}
<div class="wrapper {{ block.classes ? block.classes : '' }}"  
	{{ block.id ? 'data-' ~ block.id : '' }}
	data-width="{{ fields.width }}"
	data-background-color="{{ fields.background_color }}">
	<h1>InnerBlocks</h1>
	<InnerBlocks  />
</div>
```

If I specify:
```
Mode: preview
```
I get the InnerBlocks

If I specify
```
Mode: edit
```
I get the ACF fields.

But I can never access both at the same time.  I also tried with a separate -preview twig file with the same results.

So to sum up:
**Is there a way to get both InnerBlocks and ACF fields accessible in the Block editor on a single block?**

**Comment by palmiak** [comment]
Hi - I don't fully follow - could you show a video or something?

**Comment by thisanimus** [comment]
I see ACF fields are accessible in the block pane sidebar.  Closing.
